lib/optimizer/src/TransformationEngine.php

```diff
@@ -31,13 +31,13 @@ final class TransformationEngine
     /**
      * Instantiate a TransformationEngine object.
      *
-     * @param Configuration    $configuration Configuration data to use for setting up the transformers.
+     * @param Configuration    $configuration Optional. Configuration data to use for setting up the transformers.
      * @param RemoteGetRequest $remoteRequest Optional. Transport to use for remote requests. Defaults to the
```

Comment:
Is it useful to indicate `null` can be supplied here?

```suggestion
     * @param Configuration|null    $configuration Optional. Configuration data to use for setting up the transformers.
     * @param RemoteGetRequest|null $remoteRequest Optional. Transport to use for remote requests. Defaults to the
```